contracts/RenPool.sol

```diff
@@ -1,35 +1,43 @@
 pragma solidity ^0.8.0;
 
-import "OpenZeppelin/openzeppelin-contracts@4.0.0/contracts/token/ERC20/ERC20.sol";
+import "./RenToken.sol";
+
 
 contract RenPool {
-    ERC20 public renToken;
+    RenToken public renToken;
     address public owner; // This will be our address, in case we need to destroy the contract and refund everyone
```

Comment:
We should not rely on the RenToken but on the ERC20 standard. The RenToken is a construction we added just to be able to test the RenPool functionality again an instance of a ERC20 token.